x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/upgrade_review_algorithms.md

```diff
@@ -0,0 +1,246 @@
+# Diff Algorithms for `upgrade/_review` Endpoint
+
+This is a test plan for the `upgrade/_review` endpoint diff algorithms that are a part of the larger prebuilt rules customization feature. These algorithms determine what fields get returned when a user makes an API request to review changes as a part of the rule update process and determine what version of those fields should be displayed by the UI.
+
+Status: `in progress`.
+
+## Table of Contents
+
+- [Useful information](#useful-information)
+  - [Tickets](#tickets)
+  - [Terminology](#terminology)
+  - [Assumptions](#assumptions)
+  - [Non-functional requirements](#non-functional-requirements)
+  - [Functional requirements](#functional-requirements)
+- [Scenarios](#scenarios)
+
+  - [Rule field doesn't have an update and has no custom value - `AAA`](#rule-field-doesnt-have-an-update-and-has-no-custom-value---aaa) -[**Scenario: `AAA` - Rule field is any type**](#scenario-aaa---rule-field-is-any-type)
```

Comment:
This is incorrectly formatted so it looks like this:

![image](https://github.com/elastic/kibana/assets/5354282/3c16d5b0-de86-47d4-ac8b-f7229c69d39c)

"-[**Scenario: `AAA` - Rule field is any type**]" is missing as space after the "-" in order to consider it as part of an unordered list. Also, it should be on its own line below, with an indentation.

Same for ABA.